src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -48,8 +48,13 @@
 class PylintPartition:
     root_targets: FrozenOrderedSet[Target]
     closure: FrozenOrderedSet[Target]
+    resolve_description: str
     interpreter_constraints: InterpreterConstraints
 
+    def description(self) -> str:
+        ics = str(sorted(str(c) for c in self.interpreter_constraints))
+        return f"{self.resolve_description}, {ics}" if self.resolve_description else None
```

Comment:
This should probably still render the ICs if there are any, right?

Reply:
Oops! Yes